potbuild.sh

```diff
@@ -65,6 +65,14 @@ for file in $FLAVOUR_FILES; do
   fi
 done
 
+# searching for jail-based pot images
+NOMADCHECK=$(grep "^RUNS_IN_NOMAD=false" "$FLAVOURS_DIR"/"$FLAVOUR"/"$FLAVOUR".sh)
```

Comment:
It would be better to check the return value of grep here (returns 1 on match).

Reply:
Quoting $FLAVOUR isn't necessary, as it's guaranteed to contain nothing strange.